exercises/hello-world/lib/example.dart

```diff
@@ -1,3 +1,5 @@
 class HelloWorld {
-  String hello([String name = "World"]) => "Hello, $name!";
+  String hello() {
+    return "Hello, World!";
+  }
```

Comment:
I don't think thus change is needed. The example.dart files are outside the scope of adding the presubmit script, in my opinion.